### PR TITLE
[GUI] Added new show/dump mnemonic UI, similar to the dumpmnemonic RPC call

### LINKED ISSUE #621: Replace Master Private Key with Mnemonic
I think it would be better if instead showing the Master Private Key, would show the Mnemonic.
dumpmnemonic=""

### PR DESCRIPTION
**Pull request by mxaddict**
Capture1

Capture2

closes #621

### CODE REVIEW

**Comment by navbuilder** [comment]
A new build of 2f713d92736e994a6fde1bae65a4874114b87c6a has completed succesfully!
Binaries available at https://build.nav.community/binaries/added-dump-mnemonic-ui

**Comment by Xa5r** [comment]
I would not call it "master mnemonic"
Just "mnemonic" or "mnemonic/seed"

File: src/qt/navcoingui.cpp [hunk 1]
```diff
@@ -499,6 +500,9 @@ void NavCoinGUI::createActions()
     exportMasterPrivateKeyAction = new QAction(tr("Show &master private key"), this);
     exportMasterPrivateKeyAction->setToolTip(tr("Show master private key"));
 
+    exportMnemonicAction = new QAction(tr("Show m&nemonic"), this);
```
Comment (mxaddict) [inline src/qt/navcoingui.cpp:503]: @Xa5r I've changed the label

**Comment by mxaddict** [comment]
> I would not call it "master mnemonic"
> Just "mnemonic" or "mnemonic/seed"

Done

**Comment by navbuilder** [comment]
A new build of 5492fa5b1e20a9d1b5995fa65a5264d627aba90a has completed succesfully!
Binaries available at https://build.nav.community/binaries/added-dump-mnemonic-ui

**Pull request merged**